ci/prepare.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Exit if anything fails AND echo each command before executing
+# http://www.peterbe.com/plog/set-ex
+set -ex
+
+if [ $# -lt 3 ]; then
+	echo "usage: $0 <db-name> <db-user> <db-pass> [db-host] [wp-version]"
+	exit 1
+fi
+
+# Set up constant
+DB_NAME=$1
+DB_USER=$2
+DB_PASS=$3
+DB_HOST=${4-localhost}
+WP_VERSION=${5-latest}
+
+WORDPRESS_SITE_DIR=${WORDPRESS_SITE_DIR-/tmp/wordpress}
+
+# Create WordPress root directory
+mkdir -p $WORDPRESS_SITE_DIR
+
+# Download WordPress
+vendor/bin/wp core download --force --version=$WP_VERSION --path=$WORDPRESS_SITE_DIR --allow-root
```

Comment:
About adding WP-CLI via Composer. I can understand doing this here for is ease of installation.

All the docs say you need WP-CLI globally installed etc to use WordHat. A long time ago I did try adding WP-CLI via Composer and running from that instance, but it didn't work very well

 I am a little concerned that someone smart might notice we're using it in Composer but requiring people to install it manually and use that global version inside the WP-CLI driver. Feels a little odd. But I guess I can ignore that and explain it as an edge-case if it comes up.

What's the consequence of adding another package to the `require-dev` section? `install` will install that by default, right? Should we update the documentation examples to tell people to run `install` in such a way that the dev dependencies aren't downloaded? (I forgot the parameter name, but i'm sure there's one).